src/NetworkPort.php

```diff
@@ -1266,8 +1266,12 @@ protected function showPort(array $port, $dprefs, $so, $canedit, $agg, $rand, $w
                             break;
                         case 126: //IP address
                             $ips_iterator = $this->getIpsForPort('NetworkPort', $port['id']);
-                            foreach ($ips_iterator as $iprow) {
-                                $output .= '<br/>' . $iprow['name'];
+                            for ($i = 0; $i < count($ips_iterator); $i++) {
+                                if ($i > 0) {
+                                    $output .= '<br />';
+                                    $ips_iterator->next();
+                                }
+                                $output .= $ips_iterator->current()['name'];
```

Comment:
This would probably be more readable.
```suggestion
                            $ip_names = [];
                            foreach ($ips_iterator as $iprow) {
                                $ip_names[] = $iprow['name'];
                            }
                            $output .= implode('<br />', $ip_names);
```